src/main/java/org/opentripplanner/updater/car_park/HslCarParkDataSource.java

```diff
@@ -18,6 +28,10 @@
 
     private GeometryFactory gf = new GeometryFactory();
 
+    private final static String UTILIZATION_URL = "https://p.hsl.fi/api/v1/utilizations";
```

Comment:
Is there any way to get this url from router-config? does it support "custom variables"?